src/js/services/link_track.js

```diff
@@ -0,0 +1,186 @@
+/* eslint-env browser */
+/* global $ */
+const SEND_COMMAND = 'send';
+const EVENT_HIT_TYPE = 'event';
+const EVENT_FIELDS = ['eventCategory', 'eventAction', 'eventLabel', 'eventValue', 'transport', 'hitCallback'];
+
+/**
+ * Compulsory fields for send event on clicking a link.
+ */
+function compulsoryFields(/* link */) {
+  return {
+    transport: 'beacon',
+  };
+}
+
+/**
+ * Returns the URL origin given a string representation of it. If the browser
+ * does not support URL, it returns href itself.
+ *
+ * @param {String} href the string containing the URL.
+ * @return {String} the origin of the URL.
+ */
+function getOrigin(href) {
+  if (URL !== undefined) {
```

Comment:
If you do not hate ternaries that much, it would be more idiomatic if done:
```javascript
function getOrigin(href) {
    return window.URL ? new URL(href).origin ? href;
}
```